x-pack/plugins/ml/public/application/timeseriesexplorer/_timeseriesexplorer_annotations.scss

```diff
@@ -76,3 +76,25 @@ $mlAnnotationRectDefaultFillOpacity: 0.05;
 .mlAnnotationHidden {
   display: none;
 }
+
+// context annotation marker
+.mlContextAnnotationRect {
+  stroke: $euiColorFullShade;
+  stroke-width: $mlAnnotationBorderWidth;
```

Comment:
For what it's worth, there are 1px `$euiBorderWidthThin` and 2px `$euiBorderWidthThick` variables, if you're interested in using them here.

Reply:
Thanks for looking it over 🙏 I think I'll keep `mlAnnotationBorderWidth` here since it's used for other parts as well but good to know we do have those variables available.